easybuild/easyconfigs/i/icc/icc-2015.5.223-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,24 @@
+name = 'icc'
+version = '2015.5.223'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_ccompxe_%(version)s.tgz']
+
+gcc = 'GCC'
```

Comment:
as pointed out by @ocaisa in #2214, this should be `GCCcore`, and `binutils` should be included as well as a dep